### DISCUSSION TITLE
Filter over select fields [Nov 03]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#4364] [Discord Threads]
> 🙋‍♂️ **dvd**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### <@815870580964196403> You can create a filter for either single-select or multi-select just akin to single line text column  
![image0](https://cdn.discordapp.com/attachments/1035180106664136724/1035214219978752031/Screenshot_2022-10-27_at_9.00.07_PM.png)  
#  
> 🙋‍♂️ **dvd**  
#### yes but that would be great to click on various options instead of typing them in. I believe airtable and notion offer this possibility  
#  
> 👨‍🦲 **dstala**  
#### ah, ok. that's not supported at the moment.  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1035180106664136724)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
